src/components/settings-row/settings-row.test.js

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import SettingsRow from "./settings-row.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import { getDataElementByValue } from "../../../cypress/locators";
+import {
+  settingsRowPreview,
+  settingsRowChildren,
+  settingsRowDescription,
+} from "../../../cypress/locators/settings-row/index";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+context("Tests for SettingsRow component", () => {
+  describe("should check SettingsRow component properties", () => {
+    it.each(testData)(
+      "should check %s as title for SettingsRow component",
+      (title) => {
+        CypressMountWithProviders(<SettingsRow title={title} />);
+        getDataElementByValue("title").should("have.text", title);
+      }
+    );
+
+    it.each(testData)(
+      "should check %s as description for SettingsRow component",
+      (description) => {
+        CypressMountWithProviders(
+          <SettingsRow description={description} title="Title" />
+        );
+        settingsRowDescription().should("have.text", description);
+      }
+    );
+
+    it.each([true, false])(
+      "should check when divider property is %s for SettingsRow component",
+      (bool) => {
+        CypressMountWithProviders(
+          <SettingsRow divider={bool} description="Description" title="Title">
+            Content for settings
+          </SettingsRow>
+        );
+        if (bool === true) {
```

Comment:
Should you also be asserting here that the divider is visible, and then not visible when the prop is not set?